Makefile

```diff
@@ -1,85 +1,122 @@
+	\t make config-db 		| configure misp-db via misp-robot\n \
+	\t make config-server		| configure misp-server via misp-robot\n \
+	\t make config-proxy 		| configure misp-proxy via misp-robot\n \
+	Backup: \n \
+	\t make backup-all 		| backup all misp volumes via misp-robot\n \
+	\t make backup-server		| backup misp-server volumes via misp-robot\n \
+	\t make backup-redis		| backup misp-redis volumes via misp-robot\n \
+	\t make backup-db			| backup misp-db volumes via misp-robot\n \
+	\t make backup-proxy		| backup misp-proxy volumes via misp-robot\n \
+	\t make backup-robot		| backup misp-robot volumes via misp-robot\n \
+	\t make restore			| restore volumes via misp-robot\n \
+	\nFor testing or manul docker container only:\n \
+	\t make build-all	 		| build all misp container\n \
 	\t make build-server	 	| build misp-server\n \
 	\t make build-proxy 		| build misp-proxy\n \
 	\t make build-robot 		| build misp-robot\n \
-	\t make build-all	 		| build all misp container\n \
-	\t make security	 		| check docker security\n \
-	\t make help	 			| show help\n"
+	\t make help	 		| show help\n"
 
+# Start
+start: requirements build-config deploy configure
+	@echo "##############################\n# MISP environment is ready.\n##############################\n"
+
+####################	used as host
 # Check requirements
 requirements:
 	scripts/requirements.sh
-
-# Start Docker environment
-deploy: 
-	scripts/startup.sh
-
-# Configure Container
-configure:
-	docker exec -it misp-robot /bin/bash -c "/srv/configure_misp.sh"
+	@echo "##############################\n# Requirements for MISP environment are checked.\n##############################\n"
 
 # Build Configuration
 build-config:
-	scripts/build_config.sh
-
-# Build Docker misp-server
-build-server:
-	container/build.sh server --no-cache
-
-# Build Docker misp-proxy
-build-proxy:
-	container/build.sh proxy --no-cache
-
-# Build Docker misp-robot
-build-robot:
-	container/build.sh robot --no-cache
+	docker run --name misp-robot-init --rm -ti \
+		-v $(CURDIR):/srv/misp-dockerized \
+		dcso/misp-robot bash -c "scripts/build_config.sh"
+	@echo "##############################\n# MISP environment configuration is build.\n##############################\n"
 
-# Build all misp docker-container
-build-all:
-	container/build.sh all  --no-cache
+# Start Docker environment
+deploy: 
+	sed -i "s,myHOST_PATH,$(CURDIR),g" "./docker-compose.yml"
+	
+	docker run --name misp-robot-init --rm -ti \
+		-v $(CURDIR):/srv/misp-dockerized \
+		-v /var/run/docker.sock:/var/run/docker.sock:ro \
+		dcso/misp-robot bash -c "scripts/deploy_environment.sh"
+	@echo "##############################\n# MISP environment is deployed.\n##############################\n"
 
 # delete all misp container, volumes and images
 delete:
-	bash scripts/stop_remove_hard.sh
+	scripts/delete_all_misp_from_host.sh
 
-# rebuild all docker container
-rebuild:
-	scripts/stop_and_rebuild.sh
+delete-unused:
+	docker system prune
```

Comment:
Did we really want to delete all unused stuff e.g. things that belongs to another images or container than misp without a user prompt?

Reply:
There is a user prompt from docker itself.

Reply:
from command:
docker system prune
WARNING! This will remove:
        - all stopped containers
        - all networks not used by at least one container
        - all dangling images
        - all build cache
Are you sure you want to continue? [y/N]